Tribler/Test/Core/test_sqlitecachedbhandler_torrents.py

```diff
@@ -224,10 +230,14 @@ def test_getCollectedTorrentHashes(self):
 
     @blocking_call_on_reactor_thread
     def test_freeSpace(self):
+        # Manually set the torrent store because register is not called.
+        self.session.lm.torrent_store = LevelDbStore(self.session.get_torrent_store_dir())
+        # print self.session.lm.torrent_store
         old_res = self.tdb.getNumberCollectedTorrents()
         self.tdb.freeSpace(20)
         res = self.tdb.getNumberCollectedTorrents()
-        self.assertEqual(old_res, res)
+        self.session.lm.torrent_store.close()
+        self.assertLess(res, old_res)
```

Comment:
Maybe you should check that the amount left is the right one, not just less than before.